src/sdltiles.cpp

```diff
@@ -2296,6 +2296,39 @@ void update_music_volume() {
 }
 
 #ifdef SDL_SOUND
+std::unordered_map<std::string, Mix_Chunk*> unique_chunks;
```

Comment:
I think this should be `static`

Reply:
Sure, won't hurt to make it static. I don't think it's strictly necessary to do so (see other file-level global declarations within sdltiles.cpp) but I think it will prevent other files from externing it.